package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "blacklight-range-limit",
+  "version": "8.2.4-beta.1",
+  "description": "A range facet UI component plugin for blacklight",
+  "main": "app/assets/javascripts/blacklight",
```

Should this be blacklight_range_limit?
```suggestion
  "main": "app/assets/javascripts/blacklight_range_limit",
```